config/cli.go

```diff
@@ -22,7 +22,6 @@ type Cli struct {
 	MistUser                  string
 	MistPassword              string
 	MistPort                  int
-	MistHttpPort              int
```

Comment:
Will anything break if the infra is still trying to pass this param?

Reply:
Good question, I'll check on Staging and add it back in if needed

Reply:
Looks like we weren't actually using it anywhere